domains/sharan.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Sharan's website",
+  "repo": "https://github.com/Vasoyasharan/Mywebsite",
+  "owner": {
+    "username": "Vasoyasharan",
+    "email": "sharanvasoya@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://vasoyasharan.github.io/Mywebsite/"
```

Comment:
remove https:// and /mywebsite/ from CNAME

Reply:
thanks..